src/main/java/frc/robot/RobotContainer.java

```diff
@@ -48,15 +43,16 @@ public class RobotContainer {
     private final XboxController m_controller = new XboxController(0);
 
     // The robot's subsystems and commands are defined here...
-    private final Vision m_vision = new Vision();
-    private final DriveTrain m_driveTrain = new DriveTrain(m_vision);
+    // private final Vision m_vision = new Vision();
+    // private final DriveTrain m_driveTrain = new DriveTrain(m_vision);
+	private final Arm m_arm = new Arm();
 
     /**
      * The container for the robot. Contains subsystems, OI devices, and commands.
      */
     public RobotContainer() {
         configureButtonBindings();
-        m_driveTrain.setDefaultCommand(getDriveCommand());
+        // m_driveTrain.setDefaultCommand(getDriveCommand());
```

Comment:
If this is suppoed to work on the real robot, we 'll want to keep this. Either put it back or, if we're just testing, add a TODO comment to remind us.